src/Reflection/AST/Name.agda

```diff
@@ -10,7 +10,7 @@ module Reflection.AST.Name where
 
 open import Data.List.Base              using (List)
 import Data.Product.Properties as Prodₚ using (≡-dec)
-import Data.Word.Properties as Wₚ       using (_≟_)
+import Data.Word64.Properties as Wₚ       using (_≟_)
```

Comment:
Suggest instead avoiding the `Xₚ` suffix in favour of simply using the root module/datatype name `X` cf. #2280

Reply:
Note that it's not a change *I* am making.